cachemgr/src/main/java/com/bazaarvoice/emodb/cachemgr/core/DefaultCacheRegistry.java

```diff
@@ -110,7 +110,7 @@ public void close() {
         private final AtomicReference<Cache<String, ?>> _cache = new AtomicReference<>();
 
         private HandleImpl(String name) {
-            _name = checkNotNull(name, "name");
+            _name = Objects.requireNonNull(name, "name");
```

Comment:
From code readability prospective using calling `Objects.requireNonNull()` is still confusing. There are many places where we `com.google.common.base.Objects` is still used. So it doesn't help me to quickly identify without IDE assistance if this method belongs to Guava or Java 8. To me it clogs the code with no particular benefits. IMHO code is more readable with static imports.

Reply:
ok, changed to use static imports